fakestorage/server.go

```diff
@@ -102,6 +103,8 @@ type Options struct {
 
 	// Location used for buckets in the server.
 	BucketsLocation string
+
+	Logger *logrus.Logger
```

Comment:
hmm the thing to keep in mind is that fake-gcs-server is also used as a library, so we probably need a default logger that discards everything it receives or to wrap all calls to the logger with a nil check, which is probably not a great idea? 🤔